cli/src/commands/git.rs

```diff
@@ -361,6 +413,35 @@ pub fn git_init(
     Ok(())
 }
 
+fn cmd_git_init(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    args: &GitInitArgs,
+) -> Result<(), CommandError> {
+    let cwd = command.cwd().canonicalize().unwrap();
+    let wc_path = cwd.join(&args.destination);
+    let wc_path = file_util::create_or_reuse_dir(&wc_path)
+        .and_then(|_| wc_path.canonicalize())
+        .map_err(|e| user_error_with_message("Failed to create workspace", e))?;
+
+    git_init(
+        ui,
+        command,
+        &wc_path,
+        args.colocated,
+        args.git_repo.as_deref(),
+    )?;
+
+    let relative_wc_path = file_util::relative_path(&cwd, &wc_path);
+    writeln!(
+        ui.stderr(),
+        "Initialized repo in \"{}\"",
```

Comment:
Nit: you can use a raw string `r#"Initialized repo in "{}""#`  here, which alleviates escaping.

Reply:
Done. Thanks for the tip.